index.html

```diff
@@ -6,11 +6,21 @@
   </head>
   <body>

```

Comment:
Consider organizing the elements within the body tag. Right now, everything that is not in the footer is in the same box. There are many options for you to use to organize your data: 

http://html5doctor.com/downloads/h5d-sectioning-flowchart.png